src/component/LoginForm.vue

```diff
@@ -70,11 +71,11 @@ export default {
     }
 
     &--text {
-      background-color: rgba(255, 255, 255, 0.8);
+      background-color: transparentize(white,0.2);
```

Comment:
Same as above, save this value into a new variable.
And there's a lot of things like this I can't possibly commenting on all of them